test/unit/test_team_score.py

```diff
@@ -0,0 +1,168 @@
+from unittest.mock import patch
+import pytest
+
+from fal.controllers import team_score
+from fal.models import TeamWeeklyPoints
+
+def test_already_got_high_bonus(team_weekly_points_factory, team_factory, session):
+    team = team_factory()
+    team_weekly_points_factory(team=team, week=0)
+    team_weekly_points_factory(team=team, week=1, is_highest=1)
+    team_weekly_points_factory(team=team, week=2)
+
+    assert team_score.already_got_high_bonus(team.id, session)
+
+def test_did_not_get_high_bonus_yet(team_weekly_points_factory, team_factory, session):
+    team = team_factory()
+    for week in range(4):
+        team_weekly_points_factory(team=team, week=week)
+
+    assert not team_score.already_got_high_bonus(team.id, session)
+
+def test_get_team_scores_counts_this_week_returns_score_groups_descending_order_by_score(
+    team_weekly_points_factory,
+    team_factory,
+    session
+):
+    week = 1
+    teams = team_factory.create_batch(9)
+    team_weekly_points_factory(week=week, weekly_points=500, team=teams[0])
```

Comment:
Would it be cleaner to make these statements in a loop?
```python
for team, points in zip(teams, (500, 2000, 2000, 1500, 3000, 1000, 1000, 1000, 1000)):
    team_weekly_points_factory(week=week, weekly_points=points, team=team)
```

Reply:
I did do it this way in other tests, but for readability I felt it was important to make it clear which team got which points, and this way seemed more readable. What do u think?

Reply:
It's important to me know which team maps to which points so the assertions below make sense

Reply:
I see what you're saying, but personally I think it's easy to tell that 1500 maps to `teams[3]` based on looking at its position in the tuple. It's not a big deal either way so I'll leave it up to you.